qualtran/_infra/data_types.py

```diff
@@ -962,9 +962,15 @@ def gf_type(self):
         )
 
     def to_bits(self, x) -> List[int]:
-        """Yields individual bits corresponding to binary representation of x"""
+        """Returns individual bits corresponding to binary representation of x"""
         self.assert_valid_classical_val(x)
-        return self._quint_equivalent.to_bits(int(x))
+        ret = []
+        v = int(x)
+        for _ in range(int(self.degree)):
+            ret.append(v & 1)
+            v >>= 1
+        ret.reverse()
+        return ret
```

Comment:
Please revert this change after https://github.com/quantumlib/Qualtran/pull/1560

Reply:
done